README.md

```diff
@@ -557,6 +557,26 @@ In the "client" folder, for example, the commands that are exposed for the clien
 Functions for Github are be provided in `main/github.py`. This organization should
 be fairly straight forward to always find what you are looking for.
 
+### Drivers
+
+To test more difficult urls, we use a web driver, and you can choose between:
+
+ - [Chrome Driver](https://chromedriver.chromium.org/downloads)
+ - [Gecko Driver](https://github.com/mozilla/geckodriver/releases) (firefox)
+ 
+both to be used with selenium. This driver is optional, but will come by default with our action. To install
+it, you can download the driver at either of the links above and ensure you install selenium:
+
+```bash
+$ pip install urlchecker[selenium]
```

Comment:
so if I get this correctly  `pip install urlchecker` would install the tool as previously done, and it will work without the web-driver. Whereas this will install the library and use the web-driver?

Reply:
Yes if you do either:

```bash
$ pip install urlchecker[all]
$ pip install urlchecker[selenium]
```
It's an extra variant, and the extra dependency is installed. It means that an install without that will still work (as it does not) but the import from selenium will silently fail so we won't test things with the driver. We ask the user to explicitly install selenium like this because they also will need to download the web driver. Most users are going to use the action and won't have to think about it.

I'm working on that now - getting the driver running in the action has a few more hurdles to handle, will update over there when it's working!

Reply:
Also @SuperKogito I was having trouble pushing to Quay this morning so I moved our builds to be alongside the repo here - ghcr.io seems faster (and more nicely coupled in not requiring the extra credentials) so the only change is that the action base image will update to that (as I've done in that PR)!

Reply:
Thank you for your great work @vsoch.
I noticed the move, and as long as it works it is fine with me ;) Plus most dockers I know use ghcr.io so I think it should be good :)

Reply:
My pleasure @SuperKogito! I get so much joy from working on this little collaboration, it's projects like this that make me love open source, much more than "all that work stuff" :laughing: